CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+cmake_minimum_required(VERSION 3.16)
+project(NLCL)
+
+set(CMAKE_CXX_STANDARD 17)
+set(CMAKE_CXX_FLAGS "-lm -Wall -Werror -pedantic-errors -O0")
+
+set(SOURCES main.cpp forest.cpp)
```

Comment:
forest.cpp is not a part of testForest executable, it is part of NLCL library. testForset executable must link with this library.